tests/index/test_index_search_meilisearch.py

```diff
@@ -1,5 +1,7 @@
 # pylint: disable=invalid-name
 
+from typing import Counter
```

Comment:
Importing `Counter` from `typing` has been depreciated in Python 3.9+

```suggestion
from collections import Counter
```

Reply:
Ah whoops, should upgrade my python version, good catch